components/brave_new_tab_ui/containers/newTab/settings/solidColorChooser.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+import { StyledCustomBackgroundSettings } from '../../../components/default'
+import NavigateBack from '../../../components/default/settings/navigateBack'
+import SolidColorBackgroundOption from './solidColorBackgroundOption'
+import { solidColorsForBackground } from '../../../data/colors'
+import { getLocale } from '$web-common/locale'
+interface Props {
```

Comment:
Optional nit: Newline after imports (optional because I don't think we have a lint rule, so it's fine)